sources/packages/backend/apps/workers/src/controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -45,7 +50,7 @@ export class ProgramInfoRequestController {
     try {
       const application = await this.applicationService.getApplicationById(
         job.variables.applicationId,
-        { loadDynamicData: false },
+        { loadDynamicData: true },
```

Comment:
IMO it would be better to update the worker method name alone here. 

<img width="401" height="87" alt="image" src="https://github.com/user-attachments/assets/72b9f420-85cb-4ad9-bdb3-07261d4e2d55" />